code/modules/reagents/machinery/mortar.dm

```diff
@@ -70,9 +24,11 @@
 	if (crushable)
 		to_chat(user, "<span class ='warning'>There's already something inside!</span>")
 		return 1
-	if (!is_type_in_list(O, blend_items) && !is_type_in_list(O, juice_items))
-		to_chat(user, "<span class ='warning'>You can't grind that!</span>")
-		return ..()
+
+	//There used to be a check here to only allow certain whitelisted things to be put inside the mortar. Now we just check size. God help us.
```

Comment:
You do realize I'd be ok with this if I didn't know that you did this purely because you half-assed the refactor and didn't bother to reimplement it.

Sorry, but this is bus station all over again.